export-catalog.php

```diff
@@ -2,32 +2,32 @@
 
 require_once __DIR__.'/../../config/config.inc.php';
 require_once __DIR__.'/oyst.php';
-require_once __DIR__ . '/external/autoload.php';
 
-// Prepare Product Service
-$oyst = new Oyst();
 $context = Context::getContext();
 Context::getContext()->currency = new Currency(Configuration::get('PS_CURRENCY_DEFAULT'));
-$productRepository = new ProductRepository(Db::getInstance());
 
+// Prepare Product Service
+$oyst = new Oyst();
+
+// Get Api client
+$oystCatalogAPI = OystApiClientFactory::createCatalogApiClient(
+    $oyst->getApiKey(),
+    'PrestaShop-'.$oyst->version
+);
+
+// Get PrestaShop requirement for dependencies
+$productRepository = new ProductRepository(Db::getInstance());
 $exportProductService = new ExportProductService(
     $context,
     $productRepository,
     $oyst
 );
 
-// Prepare Api
-$conf = new OystProductApiConfigurationLoader(new Yampee_Yaml_Parser());
-$conf
-    ->load()
-    //->setEnvironment(Configuration::get('OYST_ENV'))
-;
-
-$oystCatalogAPI = new OystCatalogAPI($conf, Configuration::get('FC_OYST_API_KEY'), 'PrestaShop-'.$oyst->version);
-
 // Let's go !
 $exportProductController = new ExportProductController($exportProductService, $oystCatalogAPI);
-$exportProductController->run();
+$result = $exportProductController->run();
+
+dump($result);
```

Comment:
Cause it's a debug file to try out the process without using a request. This will be deleted before the merge.